.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-13]
+        os: [ubuntu-latest, windows-latest, macos-12]
```

Comment:
Would probably make sense to leave this as -13, but then...

Reply:
I'm leaning toward leaving as 12, just because the general advice seems to be "build on the oldest system you want to maintain compatibility with". I haven't thought about it too hard though.

Reply:
Nowadays their instructions use macos-13

https://cibuildwheel.pypa.io/en/stable/setup/#github-actions

This is probably because setting the build target should ensure compat, so no harm (and some possible benefits, e.g., newer, better compilers and software?) from using a newer OS.